backend/benefit/applications/enums.py

```diff
@@ -18,14 +18,25 @@ def is_editable_status(cls, user, status):
         if not user.is_authenticated:
             return False
         elif user.is_handler():
-            return cls.is_handler_editable_status(status)
+            return cls.is_handler_editable_status(status, None)
         else:
             return cls.is_applicant_editable_status(status)
 
     @classmethod
-    def is_handler_editable_status(cls, status):
+    def is_handler_editable_status(cls, status, new_status=None):
         if status not in cls.values:
             raise ValueError(_("Invalid application status"))
+        if new_status is not None and new_status not in cls.values:
+            raise ValueError(_("Invalid application status change"))
```

Comment:
For uniformity perhaps having the "is not None" handling would be good to have in both if statements or not in either. To me it looks it's not needed at all because cls.values does not contain None and thus "if new status is not None and new_status not in cls.values" is equal to "new_status not in cls.values".

Reply:
Hmm... come to think of Zen of Python:
> Explicit is better than implicit

So having `if new_status is not None` combined with the default value assignment `new_status=None` in the function declaration makes it *explicit that the default value is not accepted*. Because of this argument I accept this code as it is.

Reply:
new_status can be None, in the case that status is not being changed.
status on the other hand can not be None
Added comment to clarify